src/utils/make-string.js

```diff
@@ -1,22 +1,28 @@
 /** @import {Quote} from "./get-preferred-quote.js" */
 
 // Matches _any_ escape and unescaped quotes (both single and double).
-const REGEX = /\\?(?<quote>["'])/gu;
+const REGEX = /(\\.)|(["'])/gsu;
```

Comment:
@liuxingbaoyu

Do you understand, why I can't change to `/(\\["'])|(["'])/gu` to only match quote?

Change to `/(\\["'\\])|(["'])/gu` seems work, but I can't understand.

Reply:
I believe the reason previous versions used `.` was because the authors used `\.` to pair quotation marks.